SerchOddElements.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Text;
+namespace DEV_2
+{
+    class SerchOddElements
+    {
+        public string enteredString;
+        public SerchOddElements(string enteredSymbol)
+        {
+            enteredString = enteredSymbol;
+        }
+        /// <summary>
+        /// Serch elements with odd ideks in a string

```

Comment:
The method not only search but create string, but this is not obvious from the name